frontend/public/components/utils/_name-value-editor.scss

```diff
@@ -7,18 +7,44 @@
   opacity: 0
 }
 
-.pairs-list__field {
+.pairs-list__name-field {
+  padding-top: 5px;
   padding-right: 0;
 }
 
+.pairs-list__value-field {
+  padding-top: 5px;
+  padding-right: 0px;
```

Comment:
Zero is always zero regardless of unit, so drop "px" from "0px" here and below.

Reply:
sure.